### ISSUE TITLE
add external games with wine? (Linux)

### ISSUE THREAD

**Issue by kascesar** [RobLoach/love-lutris#12]
hi, it is possible launch windows games with wine ? i had a hard search for this in the web but, without luck!.
i just need know if is possible send to the terminal an order like ... 
'sudo wine64 pathtodirectorygame/game.exe' or similar! 
To the other hand, if the previos is posible, is posible add a item to lua_libretro core for launch game from there ?

I hope your can help me! thanks a lot for yor work sr!

**Comment by RobLoach** [comment]
Yes... If you're using [Lutris](https://lutris.net/), just add the game, and it should automatically appear in the UI.

If you want more control, and don't want to use Lutris, you could also try out [ChaiLove-Launcher](https://github.com/robloach/chailove-launcher).

**Comment by kascesar** [comment]
really thanks! i want to use retroarch to launch wine games! 
but ... i dont understand how to add then to ChaiLove-Launcher's menu... 
do you know some tutorial or somthing can help me ? 
PD: thanks for your request, i really appreciate it.

**Comment by RobLoach** [comment]
1. Install ChaiLove https://github.com/libretro/libretro-chailove/tree/dev#installation
2. Download the launcher https://github.com/RobLoach/ChaiLove-Launcher/releases/tag/1.0.0
3. Update [menu.json](https://github.com/RobLoach/ChaiLove-Launcher/blob/master/menu.json) with the entries you want
4. In Retroarch... Load Content, load main.chai

**Comment by kascesar** [comment]
i did this in the menu.json:
{
	"Steam": "steam &",
	"SF4": "wine64 /home/cesar/juegos/SuperStreetFighterIV/SSFIV.exe"
}

And y have request this when i press SF4 in chail's menu (of course from retroarch, after charge main.chai):

[ChaiLove] [system] love.system.execute("wine64 /home/cesar/juegos/SuperStreetFighterIV.exe")
sh: wine64: command not found
[ChaiLove] [system] Failed to execute wine64 /home/cesar/juegos/SuperStreetFighterIV.exe

PD: when i try to launch steam from chailove, i have the same error message. 
     but when i put manually the comand "steam &", steam runs normally!.
 
i need chai-script installed ? or something like this? some library ... 
I hope your can help me! 
Thnaks!

**Comment by RobLoach** [comment]
Nice work! Glad you got the launcher running.

To get Super Street Fighter 6 working, you may need to make sure it's installed....

```
sudo apt-get install wine64
```

**Comment by kascesar** [comment]
yep wine is installed, but i need to execute with sudo: 
"sudo wine64 /home/cesar/juegos/SuperStreetFighterIV/SSFIV.exe"
now i did a .sh script file with this: 

#!/bin/bash
sudo wine64 ~/juegos/SuperStreetFighterIV/SSFIV.exe
xdotool key Return
xdotool key c 1 e 2 s 3 a 4 r 5 1 2 3 9 1 2
xdotool key Return

but, the console said this u.u (c1e2s3 ... etc, is mi pasword) :

[ChaiLove] [system] love.system.execute("/home/cesar/sf4.sh")
/home/cesar/sf4.sh: line 2: sudo: command not found
/home/cesar/sf4.sh: line 3: xdotool: command not found
/home/cesar/sf4.sh: line 4: xdotool: command not found
/home/cesar/sf4.sh: line 5: xdotool: command not found
[ChaiLove] [system] Failed to execute /home/cesar/sf4.sh

some idea ?

**Comment by RobLoach** [comment]
Shouldn't need to use sudo:
https://wiki.winehq.org/FAQ#Should_I_run_Wine_as_root.3F

If you don't have access to it, try a `chmod -R 777 ~/juegos/SuperStreetFighterIV`.

If you have [Lutris](https://lutris.net/) installed, it would be `lutris lutris:rungame/super-street-fighter-iv-arcade-edition`

**Comment by kascesar** [comment]
I try tow things: 
i put this into the 'menu.json':
 
{
	"Steam": "sh ~/steam.sh",
	"SF4": "/home/cesar/sf4.sh",
	"The King of Fighters XIII": "lutris lutris:rungame/the-king-of-figthers-xiii"
}
but, i still have the previous error.
And i try this to: 

{
	"Steam": "sh ~/steam.sh",
	"SF4": "/home/cesar/sf4.sh",
	"The King of Fighters XIII": "~/kofxiii.sh"
}
with sh script :
#!/bin/bash
lutris lutris:rungame/the-king-of-figthers-xiii 

For other side, when i put this command in the console the game launch with normality.
Are the .sh script writed the rigth way?
u.u
